netutils/config/parser.py

```diff
@@ -461,20 +462,20 @@ def is_banner_one_line(config_line):
             return False
         return True
 
-    def is_banner_start(self, line):
+    def is_banner_start(self, line: str) -> bool:
         """Determine if the line starts a banner config."""
         state = super(CiscoConfigParser, self).is_banner_start(line)
         if state:
             self.banner_end = line
         return state
 
     @property
-    def banner_end(self):
+    def banner_end(self) -> str:
         """Demarcate End of Banner char(s)."""
-        return self._banner_end
+        return self._banner_end  # type: ignore
```

Comment:
`self._banner_end` evaluates to `Optional[str]` if I remove this ignore comment, therefore we either
- perform explicit Optional handling in this property or
- return an `Optional[str]` from this